backend/src/Request/Admin/StoreOwnerBranch/StoreOwnerBranchUpdateByAdminRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Request\Admin\StoreOwnerBranch;
+
+class StoreOwnerBranchUpdateByAdminRequest
```

Comment:
missing fields hint types

Reply:
I defined explicity the type of some fields like "id", meanwhile, for other fields I used just annotations because they may take value or not, like "city" field.